pydoctor/astbuilder.py

```diff
@@ -512,6 +512,8 @@ def _handleModuleVar(self,
             
             if is_constant(target, obj):
                 self._handleConstant(obj=obj, value=expr, lineno=lineno)
+                if isinstance(obj.annotation, ast.Subscript):
```

Comment:
I think this is incorrect: it is possible for `is_constant()` to return true based on the naming convention, in which case the type being subscripted could be for example `List` or `Mapping` instead of `Final`.

Reply:
Indeed, thanks. Anyway it's not displaying the right way as you can see here: https://pydoctor--397.org.readthedocs.build/en/397/docformat/epytext/epytext_demo.demo_epytext_module.html#lang ...